packaging/systemd/microshift-aio.service

```diff
@@ -14,8 +14,8 @@ Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
-ExecStartPre=/bin/bash /usr/bin/microshift-aio # not autogenerated
-ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --sdnotify=conmon --cgroups=no-conmon --rm --replace -d --name microshift-aio --privileged -v /lib/modules:/lib/modules -v microshift-data:/var/lib --label io.containers.autoupdate=registry -p 6443:6443 quay.io/microshift/microshift:4.7.0-0.microshift-2021-08-31-224727-aio
+ExecStartPre=/bin/bash /usr/bin/microshift-containerized # not autogenerated
```

Comment:
@husky-parul here, rename this to `microshift-podman-volume-check` (also in #546)?